src/libs/actions/App.js

```diff
@@ -52,11 +52,11 @@ function getLocaleAndUpdate() {
         nvpNames: ONYXKEYS.NVP_PREFERRED_LOCALE,
     }).then((response) => {
         const preferredLocale = lodashGet(response, ['nameValuePairs', 'preferredLocale'], CONST.DEFAULT_LOCALE);
-        if ((currentPreferredLocale !== CONST.DEFAULT_LOCALE) && (preferredLocale !== currentPreferredLocale)) {
-            setLocale(currentPreferredLocale);
-        } else {
-            Onyx.set(ONYXKEYS.NVP_PREFERRED_LOCALE, preferredLocale);
+        if (preferredLocale === currentPreferredLocale) {
+            return;
```

Comment:
Did you mean to remove `setLocale` here?

Reply:
Yeah it was removed in `main`

https://github.com/Expensify/App/blob/8cca7fed73cf3f707d74960296df75db81d14bc7/src/libs/Navigation/AppNavigator/AuthScreens.js#L144-L152